packages/flutter_test/lib/src/binding.dart

```diff
@@ -611,6 +642,26 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   late StackTraceDemangler _oldStackTraceDemangler;
   FlutterErrorDetails? _pendingExceptionDetails;
 
+  _MockMessageHandler? _announcementHandler;
+  List<CapturedAccessibilityAnnouncement>? _announcements;
+
+  /// Returns a list of all the accessibility announcements made by the Flutter
+  /// framework since the last time this function was called.
+  ///
+  /// It's safe to call this when there hasn't been any announcements; it will return
+  /// null in that case.
+  List<CapturedAccessibilityAnnouncement>? takeAnnouncements() {
+    assert(inTest);
+    final List<CapturedAccessibilityAnnouncement>? announcements = _announcements;
+    _announcements = null;
+    return announcements;
+  }
+
+  /// Returns the most recent announcement made by the Flutter framework.
```

Comment:
The docs should also explain what it means when this method returns null

Reply:
Changed the function to return an empty list in that case. Do you still have concerns with the documentation?